### PR TITLE
Friendlier post-setup page

### PR DESCRIPTION
**Pull request by johnnyaug**
Before:
image

After
image

### CODE REVIEW

**Review by itai-david — APPROVED** [review]
Thanks for this upgrade. New page looks much better :-)
Approving as not to block, but *if* you need someone to review js code, I would wait for another approval

File: webui/src/pages/setup/index.jsx [hunk 1]
```diff
@@ -54,48 +54,57 @@ server:
...
-                                </tr>
-                                </tbody>
-                            </Table>
-                            <Card.Text>
-                                Download the initial client configuration under your <code>$HOME/.lakectl.yaml</code> and keep a copy of the data for your login into the system<br/>
-                            </Card.Text>
-                            <Alert variant="warning">
+                            <div className={"ml-2 row mt-4"}>
+                                <div className={"col-3"}>Access Key ID:</div>
+                                <div className={"col-7"}><code>{setupData.access_key_id}</code> &#160;&#160;<ClipboardButton className={"copy-button"} variant="outline-dark" text={setupData.access_key_id} tooltip="Copy"/></div>
```
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:68]: Why not push the `variant="outline-dark"` to the css if it's shared between the 2 usages?

File: webui/src/pages/setup/index.jsx [hunk 2]
```diff
@@ -54,48 +54,57 @@ server:
...
+                            </div>
+                            <Alert className={"mt-4"} variant="warning">
                                 This is the <strong>only</strong> time that the secret access keys can be viewed or downloaded. You cannot recover them later.
                             </Alert>
-                            <hr/>
-                            <Button variant="success" href={downloadContent} taget="_blank" download="lakectl.yaml"><DownloadIcon />Download Configuration </Button>
-                            {' '}
-                            <Button variant="link" onClick={() => router.push({pathname: "/auth/login", query: {next}})}>Go To Login</Button>
+                            <div className={"mt-4"} style={{textAlign: "center"}}>
+                                <Button className={"p-2 pl-3 pr-3 after-setup-btn"} onClick={() => window.open("/auth/login", "_blank")}>Go To Login</Button>
```
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:78]: This changes the current behaviour by creating a new tab, right?

File: webui/src/pages/setup/index.jsx [hunk 3]
```diff
@@ -54,48 +54,57 @@ server:
...
+                        </Card>
+                    </Col>
+                </Row></>
         );
     }
     return (
         <Row>
             <Col md={{offset: 2, span: 8}}>
-               <Card className="setup-widget">
+                <Card className="setup-widget">
```
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:107]: Undo

File: webui/src/styles/globals.css [hunk 4]
```diff
@@ -314,19 +314,13 @@
...
   font-size: 0.7rem;
 }
 .table .change-entry-row:hover .change-entry-row-actions .btn-link,
 .table .tree-entry-row:hover .change-entry-row-actions .btn-link,
-.table .tree-entry-row:focus .change-entry-row-actions .btn-link,
-.actions-runs-list .table tbody tr:hover td .clipboard-copy button ,
-.actions-runs-list .table tbody tr:focus td .clipboard-copy button ,
-.actions-runs-list .table tbody tr:hover td .run-filter .link ,
-.actions-runs-list .table tbody tr:focus td .run-filter .link {
+.table .tree-entry-row:focus .change-entry-row-actions .btn-link {
```
Comment (itaiad200) [inline webui/src/styles/globals.css:323]: Are these changes related?

File: webui/src/lib/components/controls.jsx [hunk 5]
```diff
@@ -218,7 +218,7 @@ export const TooltipButton = ({ onClick, variant, children, tooltip, className="
     );
 };
 
-export const ClipboardButton = ({ text, variant, onSuccess, icon = <ClippyIcon/>, onError, tooltip = "Copy to clipboard"}) => {
+export const ClipboardButton = ({ text, variant, onSuccess, icon = <ClippyIcon/>, onError, tooltip = "Copy to clipboard", ...rest}) => {
```
Comment (itaiad200) [inline webui/src/lib/components/controls.jsx:221]: I don't understand where you're passing the `...rest`.

File: webui/src/pages/setup/index.jsx [hunk 6]
```diff
@@ -54,48 +54,57 @@ server:
...
                         </Card.Body>
                     </Card>
                 </Col>
             </Row>
+                <Row>
+                    <Col md={{offset: 2, span: 8}}>
+                        <Card className="mt-5">
+                            <Card.Body className={"after-setup-card"}>
+                                <h5>Configure lakectl</h5>
+                                <div>Use the command-line tool to perform Git-like operations on your data. Save the configuration file under <code>~/.lakectl.yaml</code>:</div>
```
Comment (itaiad200) [inline webui/src/pages/setup/index.jsx:89]: Minor: path is valid for POSIX only.

**Review by itaiad200 — APPROVED** [review]
Thanks for this improvement. Mainly questions and minor comments.

**Review by peacing — APPROVED** [review]

File: webui/src/pages/setup/index.jsx (hunk 1 above)
Comment (johnnyaug) [reply webui/src/pages/setup/index.jsx:68]: The variant is a react attribute, you can't apply it using CSS

File: webui/src/pages/setup/index.jsx (hunk 2 above)
Comment (johnnyaug) [reply webui/src/pages/setup/index.jsx:78]: Yes, good catch! I was thinking it's better that to have the credentials still available in the previous tab.

**Pull request merged**